Extension/src/LanguageServer/ui.ts

```diff
@@ -83,7 +83,7 @@ export class UI {
         this.browseEngineStatusBarItem.name = localize("c.cpp.tagparser.statusbar", "C/C++ Tag Parser Status");
         this.browseEngineStatusBarItem.text = "";
         this.browseEngineStatusBarItem.tooltip = localize("discovering.files.tooltip", "Discovering files");
-        this.browseEngineStatusBarItem.command = "";
+        this.browseEngineStatusBarItem.command = undefined;
```

Comment:
Does this need to be explicitly set to undefined, or can we just remove these assignment lines?

Reply:
Yeah, I removed a few unnecessary initialization lines.